pandas/core/series.py

```diff
@@ -3507,7 +3507,11 @@ def isin(self, values):
         5    False
         Name: animal, dtype: bool
         """
-        result = algorithms.isin(com._values_from_object(self), values)
```

Comment:
I wonder if the `_values_from_object` can be moved to `algorithms.isin`? Then this could just be `result = algorithms.isin(self, values)`

Reply:
yes let's try to do this, @ma3axaka can you make this change